frontend/src/app/jobsEdit/jobsEdit.component.ts

```diff
@@ -94,7 +94,8 @@ export class JobsEditComponent implements OnInit {
       'endOfPublication': this.job.endOfPublication.unix()
     }, {withCredentials: true}).subscribe((instance: any) => {
       this.job.id = instance.id;
-      this.job.approved = false;
+      this.job.email = instance.email;
+      this.job.approved = instance.approved;
```

Comment:
```suggestion
      this.job.approved = false;
```
Hesch du das absichtlich gänderet?

Reply:
Ja, damit uf em frontend immer das azeigt wird, wo ds backend het und nid irgend was